CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.10.95](https://github.com/brave/brave-browser/releases/tag/v1.10.95)
+
+- Disabled Twitter support in the rewards panel. ([#10208](https://github.com/brave/brave-browser/issues/10208))
+- Fixed "Enable bottom toolbar" being reverted in certain cases. ([#9843](https://github.com/brave/brave-browser/issues/9843))
+- Fixed ads being displayed under YouTube in certain cases. ([#10241](https://github.com/brave/brave-browser/issues/10241))
+- Fixed crash when opening new tab after disabling "Custom Homepage" theme. ([#10296](https://github.com/brave/brave-browser/issues/10296))
```

Comment:
I would add here on some screen resolutions

Reply:
do you mean like `Fixed crash on some screen resolutions when opening new tab after disabling "Custom Homepage" theme`?